microsite/data/plugins/cloud-carbon-footprint.yaml

```diff
@@ -0,0 +1,15 @@
+---
+title: Cloud Carbon Footprint
+author: Cloud Carbon Footprint
+authorUrl: https://github.com/cloud-carbon-footprint
+category: Metrics
+description: View your cloud carbon footprint by estimating energy use (kilowatt-hours) and carbon emissions (metric tons CO2e) from public cloud usage.
+documentation: https://github.com/cloud-carbon-footprint/ccf-backstage-plugin/blob/trunk/README.md
+iconUrl: https://www.cloudcarbonfootprint.org/img/logo.png
+npmPackageName: '@cloud-carbon-footprint/backstage-plugin-frontend, @cloud-carbon-footprint/backstage-plugin-backend'
```

Comment:
This should be a single package. Probably best to link to the frontend package here as the entrypoint